tools/coqdep_common.ml

```diff
@@ -198,17 +198,25 @@ let absolute_file_name basename odir =
   let dir = match odir with Some dir -> dir | None -> "." in
   absolute_dir dir // basename
 
+let ops os =
+  match  os with
+  | Some s -> String.concat "" ("Some"::[s])
+  | None -> "None"
```

Comment:
Likely this will give a warning due to bad comment placement. Also, this is almost `Option.cata`.